docs/02.API-REFERENCE.md

```diff
@@ -603,6 +603,29 @@ typedef struct
 
 - [jerry_get_memory_stats](#jerry_get_memory_stats)
 
+## jerry_external_call_info_t
+
+**Summary**
+
+Call related information passed to [jerry_external_handler_t](#jerry_external_handler_t).
+
+**Prototype**
+
+```c
+typedef struct jerry_external_call_info_t
```

Comment:
What do you think about this name? Shall I shorten it to `jerry_call_info_t`?

Reply:
I second the `jerry_call_info_t`.

Reply:
If we are at it, shall I rename jerry_external_handler_t to jerry_native_handler_t ?